config/settings.py

```diff
@@ -37,6 +37,7 @@
     CORS_ALLOWED_ORIGINS=list,
     CSRF_TRUSTED_ORIGINS=list,  # https://gidd-idmc.dev.datafriendlyspace.org
     TIME_ZONE=(str, 'Asia/Kathmandu'),
+    DOMAIN_URL=(str, 'http://localhost:7000'),
```

Comment:
Let's use APP_DOMAIN instead of `_URL` as this is not a URL.
```suggestion
    APP_DOMAIN=(str, 'http://localhost:7000'),
```